pysat/tests/test_instrument.py

```diff
@@ -539,51 +574,64 @@ def test_filenames_load_out_of_order(self):
                                stop_fname=check_fname)
         estr = '`stop_fname` must occur at a later date '
         assert str(err).find(estr) >= 0
+        return
 
     def test_next_filename_load_default(self):
-        """Test next day is being loaded (checking object date)."""
+        """Test next day loads when invoking `.next()`."""
```

Comment:
```suggestion
        """Test next day loads when invoking `.next()` starting with filename."""
```

Reply:
There is a test above that does almost the same thing as here, test_next_fname_load_default, except it uses the first filename from self.testInst.files.